tests/utilities/test_edges.py

```diff
@@ -3,7 +3,7 @@
 from prefect import Flow, Task, task
 from prefect.utilities import edges, tasks
 
-ALL_ANNOTATIONS = {edges.unmapped, edges.mapped, edges.flatten}
+ALL_ANNOTATIONS = [edges.unmapped, edges.mapped, edges.flatten]
```

Comment:
why did we change this?

Reply:
The commit message explains it a little :P but a `set` is unordered so when tests are loaded in parallel pytest will throw an error saying the workers detected different tests